library/src/main/java/com/novoda/downloadmanager/DownloadBatch.java

```diff
@@ -370,6 +373,28 @@ void delete() {
                          + ", should be deleting");
     }
 
+    private void deleteDownloadDirs() {
+        BatchStorageRoot batchStorageRoot = BatchStorageRoot.with(() -> downloadBatchStatus.storageRoot(), downloadBatchStatus.getDownloadBatchId());
+        File batchRootDir = new File(batchStorageRoot.path());
+        if (batchRootDir.exists()) {
+            deleteDir(batchRootDir);
+        }
+    }
+
+    private void deleteDir(File batchRootDir) {
```

Comment:
Avoid abbreviations to prevent confusion. 
```suggestion
    private void deleteDirectory(File batchRootDirectory) {
```

Reply:
ok